docs/csvmodels.md

```diff
@@ -219,3 +219,33 @@ class AlbumCsv(importers.CsvModel):
             )
             obj.artist = musician
 ```
+
+# Specials Fields
+## DateField
+DateField allows us to go from a text format to a date format. It has a variable that allows us to define how it will be formatted. This variable is called DEFAULT_DATE_FORMAT and we can modify it when we initialize the DateField class using the var 'strptime'
+
+```
+f = DateField(strptime='%d-%m%Y')
+```
+
+The default value of DEFAULT_DATE_FORMAT is
+```
+f = DateField()
+f.DEFAULT_DATE_FORMAT == '%Y-%m-%d'
+```
+
+## IgnoredField
+IgnoredField we use it to not take any value because it appears in a file and let it be the database itself that introduces a default value.
+This Field, if the model is well defined, should not be used
```

Comment:
So why does this field exists? Could you please define a use case including a code sample?

Reply:
When we had the old version of importer, this field was useful because the importer catched columns if match the name of the column with the model.
Actually if you not define explicitly the name of the column, this column is totally ignored.
For then, my impression is that this field is deprecated now and we can remove it.

Please tell me if you want than I remove this field

Reply:
Yes, please, do it on a **new PR**.

As this work on this PR is done I'm going to merge it.

Thanks.

Reply:
@cayop #24